src/main/scala/za/co/absa/spark/commons/implicits/StructFieldImplicits.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spark.commons.implicits
+
+import org.apache.spark.sql.types._
+import scala.util.Try
+
+object StructFieldImplicits {
```

Comment:
This is not a `StructFieldImplicits` now it is `MetadataImplicits`

Reply:
Missed this one https://github.com/AbsaOSS/spark-commons/pull/21#discussion_r787661749 

So 1 object 2 implicit classes?

Reply:
Yes, since Metadata is related to StructType